packages/trader/src/AppV2/Components/Guide/guide-description-modal.tsx

```diff
@@ -11,55 +10,66 @@ import VideoPreview from './Description/video-preview';
+    const order = [
+        CONTRACT_LIST.RISE_FALL,
+        CONTRACT_LIST.ACCUMULATORS,
+        CONTRACT_LIST.MULTIPLIERS,
+        CONTRACT_LIST.VANILLAS,
+        CONTRACT_LIST.TURBOS,
+        CONTRACT_LIST.HIGHER_LOWER,
+        CONTRACT_LIST.TOUCH_NO_TOUCH,
+        CONTRACT_LIST.MATCHES_DIFFERS,
+        CONTRACT_LIST.EVEN_ODD,
+        CONTRACT_LIST.OVER_UNDER,
+    ];
 
-        const video_src = getDescriptionVideoIds(selected_contract_type, is_dark_mode_on);
-        //TODO: temporary, until we'll have ordered list, coming from contract type selection
-        const order = [
-            CONTRACT_LIST.RISE_FALL,
-            CONTRACT_LIST.ACCUMULATORS,
-            CONTRACT_LIST.MULTIPLIERS,
-            CONTRACT_LIST.VANILLAS,
-            CONTRACT_LIST.TURBOS,
-            CONTRACT_LIST.HIGHER_LOWER,
-            CONTRACT_LIST.TOUCH_NO_TOUCH,
-            CONTRACT_LIST.MATCHES_DIFFERS,
-            CONTRACT_LIST.EVEN_ODD,
-            CONTRACT_LIST.OVER_UNDER,
-        ];
+    const toggleVideoPlayer = (e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
+        clickAndKeyEventHandler(() => setIsVideoPlayerOpened(!is_video_player_opened), e);
+    };
 
-        const toggleVideoPlayer = (e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
-            clickAndKeyEventHandler(() => setIsVideoPlayerOpened(!is_video_player_opened), e);
-        };
+    const ordered_contract_list = [...AVAILABLE_CONTRACTS].sort(
+        (a, b) => order.findIndex(item => item === a.id) - order.findIndex(item => item === b.id)
+    );
 
-        const ordered_contract_list = [...AVAILABLE_CONTRACTS].sort(
-            (a, b) => order.findIndex(item => item === a.id) - order.findIndex(item => item === b.id)
-        );
+    React.useEffect(() => {
+        if (modal_ref.current) is_video_player_opened ? modal_ref.current.showModal() : modal_ref.current.close();
+    }, [is_video_player_opened]);
 
-        React.useEffect(() => {
-            if (modal_ref.current) is_video_player_opened ? modal_ref.current.showModal() : modal_ref.current.close();
-        }, [is_video_player_opened]);
-
-        return (
-            <React.Fragment>
-                <ActionSheet.Root isOpen={is_open} onClose={onClose} position='left' expandable={false}>
-                    <ActionSheet.Portal shouldCloseOnDrag>
-                        <ActionSheet.Content className='guide__wrapper--content'>
-                            <Heading.H4 className='guide__title'>
+    return (
+        <React.Fragment>
+            <ActionSheet.Root isOpen={is_open} onClose={onClose} position='left' expandable={false}>
+                <ActionSheet.Portal shouldCloseOnDrag>
+                    <ActionSheet.Content className='guide__wrapper--content'>
+                        <Heading.H4 className='guide__title'>
+                            {show_guide_for_selected_contract ? (
```

Comment:
Seems, that I refactored whole returned stuff, but I only added `show_guide_for_selected_contract` here and on line 72